Cargo.toml

```diff
@@ -164,6 +164,7 @@ task-compat = "0.1"
 cidr-utils = "0.4.2"
 pin-project = "0.4.26"
 k8s-openapi = { version = "0.9", features = ["v1_15"], optional = true }
+nats = { version = "0.8.1", optional = true }
```

Comment:
New nats version released (0.8.2), import bug fix: 

https://github.com/nats-io/nats.rs/pull/119

Reply:
Updated to 0.8.3 now